source/features/sticky-discussion-sidebar.css

```diff
@@ -1,8 +1,9 @@
 /*
 Set the sidebar container's height to make it fit its contents
 (instead of being the full height of the discussion)
+`#discussion_bucket` holds two columns: the conversation container (1st) and the sidebar container (2nd)
 */
-.discussion-sidebar {
+#discussion_bucket > :last-child {
```

Comment:
Why this change? `#partial-discussion-sidebar` should be enough

Reply:
That's what I thought at first, but it seems GitHub didn't really change anything other than removing the `.discussion-sidebar` element that last week used to contain the `#partial-discussion-sidebar`.

`#partial-discussion-siderbar`` is at the correct height,
it's the container element (the column element) that needs its height adjusted.

Reply:
![image](https://user-images.githubusercontent.com/10238474/62722516-41a72d00-ba17-11e9-96fc-432df4688204.png)

Reply:
Right, I see the issue. I think we should just drop it, it doesn't improve the design much and it looks like an afterthought for them too, since it's out of place.

The whole CSS would be:

```css
.rgh-sticky-sidebar {
	position: sticky;
	top: 60px !important; /* Matches sticky header's height */
	z-index: 26 !important;
}

/* Hides the last divider (on pull requests) */
.rgh-sticky-sidebar + .border-top:empty {
	display: none;
}
```

And the code below would be
```js
	const sidebar = select<HTMLElement>('#partial-discussion-sidebar')!;
```